instat/dlgDescribeTwoVarGraph.vb

```diff
@@ -87,33 +112,7 @@ Public Class dlgDescribeTwoVarGraph
         ' bResetSubdialog = True
     End Sub
 
-    Private Sub InitialiseDialog()
-        ucrBase.iHelpTopicID = 416
-        ucrBase.clsRsyntax.iCallType = 3
-        ucrBase.clsRsyntax.bExcludeAssignedFunctionOutput = False
-
-        ucrSelectorTwoVarGraph.SetParameter(New RParameter("data", 0))
-        ucrSelectorTwoVarGraph.SetParameterIsrfunction()
-
-        ucrReceiverMultipleTwoVar.Selector = ucrSelectorTwoVarGraph
-        ucrReceiverMultipleTwoVar.SetSingleTypeStatus(True)
-        ucrReceiverMultipleTwoVar.SetMultipleOnlyStatus(True)
-
-        ucrSecondVariableReceiver.Selector = ucrSelectorTwoVarGraph
-        ucrSecondVariableReceiver.SetParameter(New RParameter("fill", 0))
-        ucrSecondVariableReceiver.SetParameterIsString()
-        ucrSecondVariableReceiver.bWithQuotes = False
-
-        ucrSaveGraph.SetPrefix("two_var")
-        ucrSaveGraph.SetSaveTypeAsGraph()
-        ucrSaveGraph.SetDataFrameSelector(ucrSelectorTwoVarGraph.ucrAvailableDataFrames)
-        ucrSaveGraph.SetCheckBoxText("Save Graph")
-        ucrSaveGraph.SetIsComboBox()
-        ucrSaveGraph.SetAssignToIfUncheckedValue("last_graph")
-
-    End Sub
-
-    Private Sub SetRCodeForControls(bReset As Boolean)
+    Public Sub SetRCodeForControls(bReset As Boolean)

```

Comment:
Really? it is only set for the main dialog and so need not be public. The sub-dialogue has its own setrcode. So it should still be private according to my understanding